source/set-gitmodules.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#echo ========= Setting Git Submodules to specific commits (BEGIN) =========
+if [ ".git" ] then
+	git submodule update --init --recursive
+else
+	git init
+	git submodule add https://github.com/open-source-parsers/jsoncpp.git ./external/jsoncpp
+	#if /I "%ERRORLEVEL%" gtr "1" (
```

Comment:
why are there commented out error handlers? (here and at 17)

Reply:
Was a hacky solution to get me unblocked. I have added equivalent bash checks to the script we will commit.